portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,38 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  ArrayList<String> messages = new ArrayList<String>();
+
+  public DataServlet() {
```

Comment:
Thank you for the feedback--I completely agree. I had it because we were asked to hardcode values for now to practice making/fetching JSONs, but I'll definitely move the initialization to the doGet.

Reply:
This pattern is much cleaner, sorry for contradicting the instructions though.